### PR TITLE
Update plan designer compose and readme

### PR DESCRIPTION
**Pull request by athish-t**

### CODE REVIEW

**Review by veerajsk — APPROVED** [review]

**Comment by athish-t** [comment]
close and reopen to see if ci passes

**Comment by athish-t** [comment]
wow ci passes after reopening!

@abhi1287 bump for review

File: alica_designer_runtime/README.md [hunk 1]
```diff
@@ -266,17 +280,38 @@ over the element you want to remove and clicking on the "X".
...
+![import_export_fs](./doc/import_export_fs.png)
+
+This will export the plans directly to the host's filesystem.
+
+#### 3.1.2 Import
+Same as export, but click on File System -> Import. This will import plans directly from the host's filesystem.
+
+On successful import the plans should be visible in the right panel.
+
 ### 3.1 Zip-Workflow
```
Comment (abhi1287) [inline alica_designer_runtime/README.md]: 3.2?

File: alica_designer_runtime/docker-compose.yml [hunk 2]
```diff
@@ -30,9 +30,13 @@ services:
       - SOCIAL_APP_CLIENT_ID=${SOCIAL_APP_CLIENT_ID}
       - SOCIAL_APP_SECRET=${SOCIAL_APP_SECRET}
       - OAUTH_APPLICATION_URL=http://localhost:3030
+      - NATIVE_MODE=${NATIVE_MODE:-true}
```
Comment (abhi1287) [inline alica_designer_runtime/docker-compose.yml]: leave the default to false

**Review by abhi1287 — APPROVED** [review]

File: alica_designer_runtime/README.md (hunk 1 above)
Comment (athish-t) [reply alica_designer_runtime/README.md]: fixed

File: alica_designer_runtime/docker-compose.yml (hunk 2 above)
Comment (athish-t) [reply alica_designer_runtime/docker-compose.yml]: done

**Pull request merged**